src/components/overview/cart/Cart.jsx

```diff
@@ -94,6 +91,11 @@ function Cart({ skus, id }) {
 
   const range = (n, m) => [...Array(Math.min(n, m)).keys()];
 
+  const handleSaved = (e) => {
+    e.preventDefault();
+    document.getElementById('addOutfit').click();
```

Comment:
I believe this can be handled by Redux as well, I just completely forgot to tell you about the function😆. If you import `useDispatch` from `react-redux` and `addToOutfit` from `productReducer.js`, you can access a redux function to update the outfit list state. The syntax is a bit confusing, but you need a "dispatcher", which can be created by adding `const dispatch = useDispatch()`, and then you can add to the outfit by adding `dispatch(addToOutfit(id))`. I should have followed up with you about that function earlier, so I'm really sorry about that!